template/images.d.ts

```diff
@@ -5,6 +5,19 @@ declare module '*.jpeg'
 declare module '*.gif'
 declare module '*.bmp'
 declare module '*.tiff'
+
+declare module '*.module.css' {
+  const exports: { [exportName: string]: string };
+  export = exports;
+}
+declare module '*.module.scss' {
+  const exports: { [exportName: string]: string };
+  export = exports;
+}
+declare module '*.module.sass' {
+  const exports: { [exportName: string]: string };
+  export = exports;
+}
```

Comment:
should it still reside in `images.d.ts`? Should it be `styles.d.ts`

Reply:
you are right.

Reply:
you are right.